### ISSUE TITLE
Add Tags component with stories list on publicStory page

### ISSUE THREAD

**Issue by moranw** [yarindeoh/safe-home-haytiSham#106]
publicStory page (single video page) should include tags component and stories list, with the same behavior as it appears on the homepage (as explained on the figma)

**Comment by yarindeoh** [comment]
Agreed it should not be the same behavior. Needs to display geyByTags with the specific story tags.
